src/fu-hwids.c

```diff
@@ -259,7 +259,8 @@ fu_hwids_convert_string_table_cb (FuSmbios *smbios,
 	tmp = fu_smbios_get_string (smbios, type, offset, error);
 	if (tmp == NULL)
 		return NULL;
-	return g_strdup (tmp);
+	/* ComputerHardwareIds.exe seems to strip trailing spaces */
+	return g_strchomp (g_strdup (tmp));
```

Comment:
I've since found that it ignores leading space too; will fix.

Reply:
PR updated, I reimplemented a slightly safer version of g_strstrip() to avoid returning in the middle of an allocated string.